spec/controllers/followings_controller_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'spec_helper'
+
+describe FollowingsController do
+  describe "GET index" do
+    it_behaves_like "a security guard" do
```

Comment:
If you create a lot of shared example scenarios, you are going to want the descriptions to be accurate. What is the security guard doing? Is he not letting anybody in? Is he only letting in admins? Or anybody that is logged in? Your specs should be very easy to read, this one is not unfortunately.